Development.md

```diff
@@ -0,0 +1,250 @@
+# Convox Rack Development Guide
+
+Rack is open source with the goal of enabling everyone to understand the platform and to sugguest and contribute improvements.
```

Comment:
Maybe something like: `...everyone to understand the platform, provide suggestions, and contribute improvements.`

Cleans up the multiple `and`s a bit.